docs/posts/masquerade-as-the-bce-inc-home-hub-4000-on-xgs-pon-with-the-bfw-solutions-was-110.md

```diff
@@ -109,6 +109,11 @@ identifiers are available on the back label of the Home Hub 4000, color-coordina
 
 ![Home Hub 4000 label](masquerade-as-the-bce-inc-home-hub-4000-on-xgs-pon-with-the-bfw-solutions-was-110/home_hub_4000_label.webp){ class="nolightbox" id="home-hub-4000-label" }
 
+!!! note
```

Comment:
Should be info as it's informative and can be condensed into the title, but it's kinda redundant.